src/pages/learn.tsx

```diff
@@ -114,10 +123,26 @@ export default ({ data, location }: Props) => {
           {pages.slice(5)}
         </ul>
       </nav>
+      <div>
+        <article className="article-reader" dangerouslySetInnerHTML={{ __html: activePage.html }} />
+        <a 
+          href={`https://github.com/nodejs/website-prototype/edit/master/src/documentation/${activePage.relativePath}`}
+          style={{
+            color: "var(--gray9)",
+            fontWeight: 600,
+            textDecoration: "underline",
+            fontFamily: "var(--serif)",
+          }}
```

Comment:
Now: 

![screen shot 2019-02-18 at 1 09 51 pm](https://user-images.githubusercontent.com/23313266/52929331-f886f200-337e-11e9-8aac-de67a35c99c6.png)


IMHO, it's maybe better to be consistent with the article text above. So can we add `padding-left: 4.2rem;` here like the `article-reader` did ?

https://github.com/nodejs/website-prototype/blob/7e6f1ad84cfb3cbb0c782e5714535c4fc8fab714/src/components/layout.css#L378

Reply:
Hold on there is a bug. Last rebase mixed it up.